metrics/network/remote_network/README.md

```diff
@@ -81,3 +84,9 @@ $ bash remote-networking-cpu.sh -i "<interface_name>" -u "<user>" -a "<ip_addres
 
 Run the `remote-networking-qperf.sh` script in host A. The script requires a number of options to
 be specified. Run `remote-networking-qperf.sh` -h for details.
+
+```
+## Running the `direct-remote-networking-iperf3.sh` test
+
+Run the `direct-remote-networking-iperf3.sh` script in host A. The script requires a number of options to
```

Comment:
Lines 91-92 suggested rewrite: The script requires you to specify a number of options.